.travis.yml

```diff
@@ -5,14 +5,14 @@ branches:
   only:
     - master
 env:
-  - DOCKERFILE=Dockerfile CACHE=latest SCRIPT=src/scripts/coverage.sh
-  - DOCKERFILE=Dockerfile.nop11 CACHE=latest SCRIPT=src/scripts/test.sh
-  - DOCKERFILE=Dockerfile.deb-stable CACHE=latest-deb-stable SCRIPT=src/scripts/test.sh DARGS="-eBUILD_ONLY=1"
-  - DOCKERFILE=Dockerfile.deb-unstable CACHE=latest-deb-unstable SCRIPT=src/scripts/test.sh DARGS="-eBUILD_ONLY=1"
+  - DOCKERFILE=Dockerfile SCRIPT=src/scripts/coverage.sh
+  - DOCKERFILE=Dockerfile.nop11 SCRIPT=src/scripts/test.sh
+  - DOCKERFILE=Dockerfile.deb-stable SCRIPT=src/scripts/test.sh DARGS="-eBUILD_ONLY=1"
+  - DOCKERFILE=Dockerfile.deb-unstable SCRIPT=src/scripts/test.sh DARGS="-eBUILD_ONLY=1"
 services:
   - docker
 script:
   - docker pull advancedtelematic/aktualizr:$CACHE
-  - docker build --cache-from advancedtelematic/aktualizr:$CACHE -t advancedtelematic/aktualizr -f ${DOCKERFILE} .
+  - docker build -t advancedtelematic/aktualizr -f ${DOCKERFILE} .
```

Comment:
Why are these changes needed?  Is there something broken with the caching behaviour?